src/test/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/resource/ActivityMigrationResourceIntTest.kt

```diff
@@ -1,4 +1,4 @@
-@file:OptIn(ExperimentalCoroutinesApi::class, ExperimentalCoroutinesApi::class)
+@file:OptIn(ExperimentalCoroutinesApi::class, ExperimentalCoroutinesApi::class, ExperimentalCoroutinesApi::class)
```

Comment:
😆 I can't believe I didn't spot that. It's because there's a bug in Intellij where it warns that you need an @OptIn annotation even though you already have one. But when I click on the fix the warning goes away - didn't realise it's been duplicating the annotations! Thanks, I'll fix.